pkg/sql/logictest/testdata/logic_test/udf

```diff
@@ -1539,3 +1533,153 @@ l1  l2  i1  i2  s1  s2  v1  v2
 1   1   1   1   1   1   11  11
 2   2   2   2   2   2   12  12
 3   3   3   3   3   3   13  13
+
+
+subtest implicit_record_types
+
+statement ok
+CREATE TABLE imp(k INT PRIMARY KEY, a INT, b TEXT);
+INSERT INTO imp VALUES (1, 2, 'a');
+
+statement ok
+CREATE FUNCTION imp_const_tup() RETURNS imp LANGUAGE SQL AS $$
+  SELECT (11, 22, 'b')
+$$
+
+# TODO(mgartner): The type of imp_const_tup() should be imp, not record.
+query TBT
+SELECT imp_const_tup(), (11,22,'b')::imp = imp_const_tup(), pg_typeof(imp_const_tup())
+----
+(11,22,b)  true  record
```

Comment:
heh, looks like it returns `imp` instead of `record` in `fakedist-disk_test` test config.

Reply:
Weird.. Ok I'll have to debug why it's `record` in the local config.